src/components/structures/LoggedInView.js

```diff
@@ -264,17 +269,19 @@ export default React.createClass({
                 break;
 
             case PageTypes.HomePage:
-                // If team server config is present, pass the teamServerURL. props.teamToken
-                // must also be set for the team page to be displayed, otherwise the
-                // welcomePageUrl is used (which might be undefined).
-                const teamServerUrl = this.props.config.teamServerConfig ?
-                    this.props.config.teamServerConfig.teamServerURL : null;
-
-                page_element = <HomePage
-                    teamServerUrl={teamServerUrl}
-                    teamToken={this.props.teamToken}
-                    homePageUrl={this.props.config.welcomePageUrl}
-                />;
+                {
```

Comment:
ah, so this is to fix the lint error that we have a const declaration in a switch statement, because the variables ends up with the whole switch block as a scope which you don't necessarily expect. This limits the scope of the variable to the one case block.